src/Mvc/Mvc.Razor/src/RazorPage.cs

```diff
@@ -223,11 +235,15 @@ public void IgnoreSection(string sectionName)
 
             if (!PreviousSectionWriters.ContainsKey(sectionName))
             {
-                // If the section is not defined, throw an error.
-                throw new InvalidOperationException(Resources.FormatSectionNotDefined(
```

Comment:
Could we remove this exception instead? I wouldn't consider it part of the API contract and if it gets in the way, we might as well remove it.

Reply:
Removing the exception would work for me. Slight risk it would break anyone reliant on the throw for some kind of error logic?

Reply:
> Slight risk it would break anyone reliant on the throw for some kind of error logic?

That should be fine. That's not a compelling scenario.

Reply:
You want me to make the change or should we wait for any more discussion?

Reply:
Go for it 👍🏽

Reply:
Updated